### ISSUE TITLE
Favorite Count is Binding In Reverse

### ISSUE THREAD

**Issue by tjohnson1106** [tjohnson1106/saga-beta-V1#11]
When favorite count is 1 isFavorited is showing false and when zero shows true. Incrementation appears to be correct; when favorite icon is pressed we get an unhandled in react-apollo warning(app does not break) suggesting that data binding is incorrect somewhere. Since waiting on back-end implementation this takes low priority, but error is likely somewhere in the client side subscription model.

**Comment by tjohnson1106** [comment]
Fixed in V-0.4 rebuild. Closing issue.
